docs/src/main/asciidoc/spring-boot-properties.adoc

```diff
@@ -6,7 +6,6 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 = Quarkus - Accessing application properties with Spring Boot properties API
 
 include::./attributes.adoc[]
-:extension-status: preview
```

Comment:
I have no opinion about if we should make them stable or not but if we do, you should remove `include::./status-include.adoc[]` for all the extensions you want to make stable.

Reply:
Aha I did not do that. I'll push a new commit which removes this.

Reply:
Pushed a new commit removing the `include::./status-include.adoc[]`